middlewares/validateBody.js

```diff
@@ -0,0 +1,14 @@
+import { HttpError } from '../helpers/HttpError.js';
+
+// Middleware for validate body
+export const validateBody = (schema) => {
+  const func = (req, res, next) => {
+    const { error } = schema.validate(req.body);
+    if (error) {
+      next(HttpError(400, 'missing required name field'));
```

Comment:
I don't understand this message, it's only for `name` field?

Reply:
No. It's for any required field.
I will change this message.